ext/standard/http.c

```diff
@@ -360,3 +361,20 @@ PHP_FUNCTION(request_parse_body)
 	SG(request_parse_body_context).throw_exceptions = false;
 	memset(&SG(request_parse_body_context).options_cache, 0, sizeof(SG(request_parse_body_context).options_cache));
 }
+
+PHP_FUNCTION(http_get_last_response_headers)
+{
+	zend_parse_parameters_none();
```

Comment:
You should check the return value, _or_ use the macro version I think.

Reply:
Indeed, and I didn't initially what you meant .-. so now I'm also using explicit return macros.